packages/core/src/types/asyncapi2.ts

```diff
@@ -1099,12 +1086,32 @@ export const AsyncApi2Types: Record<string, NodeType> = {
 
   ServerBindings,
   ChannelBindings,
+  MessageBindings,
+  OperationBindings,
+};
+
+export const AsyncApi2Types: Record<string, NodeType> = {
+  ...AsyncApi2Bindings,
+  Root,
+  Tag,
+  TagList: listOf('Tag'),
+  ServerMap,
+  ExternalDocs,
+  Server,
+  ServerVariable,
+  ServerVariablesMap: mapOf('ServerVariable'),
+  SecurityRequirement,
+  SecurityRequirementList: listOf('SecurityRequirement'),
+  Info,
+  Contact,
+  License,
   ChannelMap,
   Channel,
   Parameter,
   ParametersMap: mapOf('Parameter'),
   Operation,
   Schema,
+  SchemaList: listOf('Schema'),
```

Comment:
should this remain consistent with other `type` files.  `NamedSchemas: listOf('Schema')` ??

Reply:
the same thing